utils/mime.go

```diff
@@ -27,6 +27,12 @@ const (
 
 	// MimeTypeSuffixLazy is used to indicate a lazy loading of data.
 	MimeTypeSuffixLazy = "lazy"
+
+	// MimeTypeTabular used to indicate tabular data, this is used mainly for filtering data.
+	MimeTypeTabular = "text/tabular"
```

Comment:
Adding @bhaney since I believe he added other custom mime types.

Standard naming convention is `x-` for experimental and `vnd.` for vendor-specific. If we're adding a mime type for "tabular" sensor data for the purposes of filtering and viewing data in a table in the UI, do you think a good name might be:

`x-application/tabular`?

Reply:
@tahiyasalam I think can make it x-application/tabular for now